src/app/main.c

```diff
@@ -242,6 +252,36 @@ main (int    argc,
 
   if (!command->fn)
     {
+      command = preview_commands;
+      while (command->name)
+        {
+          if (g_strcmp0 (command_name, command->name) == 0)
+            {
+              g_printerr ("notice: %s is a preview command and subject to change.\n",
```

Comment:
Regarding capitalization, I don't think we're very consistent.  For example, `error: Foo bar` is lowercase.  GNU commands seem to prefix with their command and use lowercase too:

```
$ rm /enoent
rm: cannot remove ‘/enoent’: No such file or directory
```

I feel like there's a distinction between "log messages" and "multi-line diagnostic output".  The "notice: " here is like a log message prefix, not part of a formatted text, in the same way `error: ` is.